src/Cake.Common/Tools/InnoSetup/InnoSetupSettings.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Cake.Core.IO;
+using Cake.Core.Tooling;
+
+namespace Cake.Common.Tools.InnoSetup
+{
+    /// <summary>
+    /// Contains settings used by the <see cref="InnoSetupRunner"/>.
+    /// </summary>
+    public sealed class InnoSetupSettings : ToolSettings
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="InnoSetupSettings"/> class with the default settings.
+        /// </summary>
+        public InnoSetupSettings()
```

Comment:
@thomaslevesque this should have thrown a StyleCop warning.  You may have had to switch to release mode in order for them to show up as errors though.  @patriksvensson has the benefit of being a walking/talking human StyleCop, and can see errors from 100 yards away 😄